### ISSUE TITLE
Question about calculating confidence score in gentestcase.py

### ISSUE THREAD

**Issue by lanbowen23** [XUSean0118/DVSNet#2]
When calculating the confidence score for the future training, the score should be calculated on current frame, but in the code below, the 'raw_output' is still the segmentation for the key frame. So is this a problem?

`wight = tf.where(tf.equal(raw_output, 255),tf.zeros_like(raw_output),tf.ones_like(raw_output))
    accuracy = tf.where(tf.equal(output, raw_output),wight,tf.zeros_like(raw_output))
    average = tf.divide(tf.reduce_sum(tf.contrib.layers.flatten(accuracy), 1),tf.reduce_sum(tf.contrib.layers.flatten(wight), 1))`

**Comment by XUSean0118** [comment]
Yeah, I fix and update the "gentestcase.py". Now, it calculates the current frame segmentation outputsbefore calculating confidence score.

**Comment by lanbowen23** [comment]
Thank you!
